frc42_dispatch/hasher/src/hash.rs

```diff
@@ -58,15 +58,15 @@ pub enum IllegalNameErr {
 impl<T: Hasher> MethodResolver<T> {
     const CONSTRUCTOR_METHOD_NAME: &'static str = "Constructor";
     const CONSTRUCTOR_METHOD_NUMBER: u64 = 1_u64;
-    const RESERVED_METHOD_NUMBER: u64 = 0_u64;
+    const RESERVED_METHOD_NUMBER: u64 = 256_u64;
```

Comment:
Since 256 is the first _available_ method number, let's name the constant `FIRST_METHOD_NUMBER` or similar.